src/style.css

```diff
@@ -0,0 +1,102 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+body {
+  box-sizing: content-box;
+  margin: 0;
+}
+
+.Search {
+  height: 70px;
+  width: 100vw;
+  margin-top: 20px;
+  margin-left: auto;
+  margin-right: auto;
```

Comment:
The form should take all width of the page (excluding default body margins) so there is no need to centalize it with margin auto - remove those